lib/rex/post/meterpreter/ui/console/command_dispatcher/extapi/adsi.rb

```diff
@@ -40,6 +43,52 @@ def name
   end
 
   #
+  # Options for the adsi_nested_group_user_enum command.
+  #
+  @@adsi_nested_group_user_enum_opts = Rex::Parser::Arguments.new(
+    "-h" => [ false, "Help banner" ],
+    "-m" => [ true, "Maximum results to return." ],
+    "-p" => [ true, "Result set page size." ]
+  )
+
+  def adsi_nested_group_user_enum_usage
+    print(
```

Comment:
Inline newlines can break in weird ways, please put each line into a print_line() call